Libplanet.Stun/Stun/Messages/StunMessage.cs

```diff
@@ -236,18 +229,23 @@ public byte[] Encode(IStunContext ctx)
             return ms.ToArray();
         }
 
+        internal static IEnumerable<Attribute> ParseAttributes(
+            IEnumerable<byte> bytes)
+        {
+            return ParseAttributes(bytes, System.Array.Empty<byte>());
+        }
+
```

Comment:
This is unnecessary in my opinion. Attributes are parsed only on incoming messages and they are always expected to have non-null `transactionId`. This is purely for the convenience of making a test code and should not be part of the API.